### DISCUSSION TITLE
Please share ideas which interfaces we need to configure for clean and sexy netflow data.

### DISCUSSION THREAD

**Discussion by ivanovuri** [akvorado/akvorado#1042] [Q&A]
Hi community,

What are the best practices to configure input/output sampling on the interface?
Here is an example network topology of mostly B2B service providers.
Uplinks and downlinks are BGP and you can see one backbone link between R1 and R2.
![image](https://github.com/akvorado/akvorado/assets/108669313/81014437-0b5c-4bb9-8d63-245aa388a4be)
What do u think.
Should I blindly enable input/output sampling on all interfaces Uplinks/Downlinks/Backbones or should I enable it only on uplinks in one direction?
Do we need backbone links, or such links are not good choice to activate netflow on?
In general, I need to analyze what type of traffic my downlinks consume.

Thanks in advance.

**Comment by vincentbernat** [comment]
If possible, it's easier to enable only on your uplinks on both egress/ingress directions (or downlinks depending on what is more important). You can also enable on everything, but be sure to always use the right filter to not count traffic twice (and not that the bidirectional view negate, so InIfBoundary = external becomes InIfBoundary != external).

**Comment by ivanovuri** [comment]
Thank Vincent.
Can you please share what best practices when to use MPLSLabels, MPLS1stLabel etc.
This is mostly for InterISP monitoring or for MPLS backbones?
I think in case of heavy services splitting to VRFs (customer, Uplinks etc) we can use this to understand labeled traffic pattern it good to be used or to subdivide L2VPN channels.
